src/main/content/_assets/js/collections.js

```diff
@@ -3,7 +3,8 @@ $(document).ready(function() {
         .then(setInstanceSelections)
         .then(handleInitialCLIAuth)
         .then(getCollectionData)
-        .then(updateCollectionView);
+        .then(getCliVersion)
+        .then(updateCliVersion);
```

Comment:
We should create a single entry point function where once we have the instance name we can shoot off multiple async requests at once instead of chaining them unnecessarily.

so for example a new function called something like handleRequests would have 2 lines in calling `getCollectionData` and `getCliVersion`

this would improve the speed of data requests especially when we start adding more and more